_test/test_mpi/test_job_dispatcher_mpiexec.m

```diff
@@ -23,11 +23,11 @@ function test_job_fail_restart(obj, varargin)
             end
         end
         function test_job_with_logs_3workers(obj, varargin)
-            if ispc % fail in sequence of tests on winwods
+            if ispc % fail in sequence of tests on windows
```

Comment:
why?

Reply:
this would be a ticket I will write when dealing with MPI framework. This puzzles me -- I can not imagine what the reason is behind this.

Reply:
This test will be re-enabled as part of #92

Reply:
on Unix yes -- it runs, but on windows, even locally it hangs up. Passed couple of random times but usually -- is reliable hang-up